library/src/main/res/values-v21/styles.xml

```diff
@@ -9,4 +26,15 @@
         <item name="android:textStyle">normal</item>
         <item name="android:fontFamily">sans-serif-medium</item>
     </style>
+
+    <style name="Chucker.TextAppearance.Value" parent="android:TextAppearance.Small">
+        <item name="android:paddingStart">16dp</item>
+        <item name="android:textIsSelectable">true</item>
+    </style>
+
+    <style name="Chucker.TextAppearance.TransactionTitle" parent="TextAppearance.AppCompat.Widget.ActionBar.Title">
+        <item name="android:gravity">top</item>
+        <item name="android:textSize" tools:ignore="SpUsage">16dp</item>
+        <item name="android:maxLines">2</item>
+    </style>
```

Comment:
Why is this needed?

Reply:
Just left it from original styles. It is used in `TransactionActivity` and `ErrorActivity` for toolbar title. Actually, I am against using dps for text sizes here. Will update as well.